UI/zip-it/src/data/mutations.tsx

```diff
@@ -18,7 +18,7 @@ function mutation(props): DocumentNode {
     } if (props.type === "confirm") {
         result = gql`
                 mutation {
-                    confirmUser(userEmail: "${props.data.userEmail}", confirmationCode: ${props.data.confirmationCode}) {
+                    confirmUser(userEmail: "${props.data.userEmail}", confirmationCode: ${props.data.confirmationCode}, }) {
```

Comment:
Can I ask why this is a mutation and not a query?

Reply:
This wasn't implemented by myself or this branch and currently resides in main.